t/data/tests/tests/boot.lua

```diff
@@ -0,0 +1,24 @@
+use("testapi")
+use("testlib")
+use("testlib", {"testfunc2"})
+local luatestlib = require 'luatestlib' -- load luatestlib.lua from ../lib/ dir
+
+function run(self)
+    testfunc1()
+    testfunc2()
+    luatestlib.testfunc3()
```

Comment:
We should probably check whether these functions are called correctly, e.g. check their return values.